python/src/aiconfig/util/params.py

```diff
@@ -243,6 +242,9 @@ def collect_prompt_references(current_prompt: Prompt, ai_config: "AIConfigRuntim
             break
 
         prompt_input = get_prompt_template(previous_prompt, ai_config)
+        if prompt_input is None:
```

Comment:
nit: comment saying something like "not all model parsers are parameterizable" would be nice, and where ryan commented